sci-libs/mpfun90/mpfun90-2023.02.26.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="High-Precision Software"
+HOMEPAGE="
+	https://www.davidhbailey.com/dhbsoftware/
+	https://github.com/APN-Pucky/mpfun90
+"
+MY_PV=$(ver_rs 1- '-')
+SRC_URI="https://github.com/APN-Pucky/mpfun90/archive/refs/tags/${MY_PV}.tar.gz"
+S="${WORKDIR}/mpfun90-${MY_PV}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="${DEPEND}"
+
+src_compile() {
+	tc-export FC
+	emake -j1 dynamic FC=${FC}
```

Comment:
Is this another Fortran one? If so then we need `fortran-2.eclass`, I think it will set `FC` automatically.

Reply:
Yes, it does. Thanks!